.github/workflows/build-and-push-container-image.yml

```diff
@@ -29,15 +37,16 @@ jobs:
 
       - name: Get Tags
         run: |
-            if [ ${{ steps.isVersioningCommit.outputs.isVersioningCommit }} ]; then
+            if ${{ steps.isVersioningCommit.outputs.IS_VERSIONING_COMMIT }}; then
                 tagVersion=${{ steps.version.outputs.VERSION }}
-                tags="octopusdeploy/nfs-server:$tagVersion,octopusdeploy/nfs-server:latest";
+                tags="octopusdeploy/nfs-server:$tagVersion,docker.packages.octopushq.com/octopusdeploy/nfs-server:$tagVersion,octopusdeploy/nfs-server:latest,docker.packages.octopushq.com/octopusdeploy/nfs-server:latest";
             else
+                : # git describe --tags produces a value matching <tag>-<commits since tag>-<8 char commit hash>
```

Comment:
`: #` for comments in `run: |` commands pinched from here: https://stackoverflow.com/a/68421196